src/main/java/nl/workingtalent/wtacademy/book/BookService.java

```diff
@@ -13,8 +17,17 @@ public class BookService {
 	@Autowired
 	private IBookRepository repository;
 
-	public List<Book> getAllBooks() {
-		return repository.findAll();
+	private final int pageSize = 5;
+
+	public Page<Book> getAllBooks(int pageNr) {
+		// Bouw een filtering + sortering
```

Comment:
Comments in deze method zijn in het Nederlands

Reply:
Verandert naar engels en deels verwijderd